### PR TITLE
magento/merchdocs#1103: Allow Reorders from customer account

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

Fix pull request fixes https://github.com/magento/merchdocs/issues/1103

## Magento release version

- [x] 2.4.x

- [ ] 2.3.x

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

## Affected documentation pages

-  https://docs.magento.com/user-guide/sales/reorders-allow.html

whatsnew
Added the 'Reorder from Storefront' and other sections to the [Allow Reorders](https://docs.magento.com/user-guide/sales/reorders-allow.html) topic.

### CODE REVIEW

**Comment by dobooth** [comment]
Hi @EvheniiaKolkhiieva. Thanks for your contribution! Can you please sign the Adobe CLA so that we may process this PR?

File: src/sales/reorders-allow.md [hunk 1]
```diff
@@ -32,3 +32,61 @@ _Customer reorder link in the Admin_
...
+
+User will be redirected to the Cart and all products are added to the Cart
+
+![]({% link images/images/shopping-cart-page.png %}){: .zoom}
+
+**Case 2.** Some/all products from the order are **non-available** for Reorder
+
+The "Reorder" button will not appear on the "My Orders" and "View Order" pages.
+
+![]({% link images/images/my-orders-view-page1.png %}){: .zoom}
```
Comment (atwixfirster) [inline src/sales/reorders-allow.md:60]: @EvheniiaKolkhiieva , please remove a space at the end of sentence.

Thank you!

File: src/sales/reorders-allow.md [hunk 2]
```diff
@@ -32,3 +32,61 @@ _Customer reorder link in the Admin_
...
+
+**Case 1.** All products from the order are available for Reorder
+
+User will be redirected to the Cart and all products are added to the Cart
+
+**Case 2.** Some/all products from the order are **non-available** for Reorder
+
+The "Reorder" button will not appear on the "My Orders" and "View Order" pages.
+
+![]({% link images/images/order-view-page.png %}){: .zoom}
```
Comment (atwixfirster) [inline src/sales/reorders-allow.md:72]: The same here.

File: src/sales/reorders-allow.md [hunk 3]
```diff
@@ -32,3 +32,61 @@ _Customer reorder link in the Admin_
...
+
+User will be redirected to the Cart and all products are added to the Cart
+
+**Case 2.** Some/all products from the order are **non-available** for Reorder
+
+The "Reorder" button will not appear on the "My Orders" and "View Order" pages.
+
+![]({% link images/images/order-view-page.png %}){: .zoom} 
+
+## Cart isn’t empty
```
Comment (atwixfirster) [inline src/sales/reorders-allow.md:74]: The same here.

File: src/sales/reorders-allow.md [hunk 4]
```diff
@@ -32,3 +32,61 @@ _Customer reorder link in the Admin_
...
+
+**Case 2.** Some/all products from the order are **non-available** for Reorder
+
+The "Reorder" button will not appear on the "My Orders" and "View Order" pages.
+
+![]({% link images/images/order-view-page.png %}){: .zoom} 
+
+## Cart isn’t empty 
+
+In case if Cart is not empty and user will click on 'Reorder' button (from “My Orders”  or “Order View” pages), the existing products will be added to the Shopping cart with the recorder products
```
Comment (atwixfirster) [inline src/sales/reorders-allow.md:76]: The same here.

File: src/sales/reorders-allow.md [hunk 5]
```diff
@@ -32,3 +32,61 @@ _Customer reorder link in the Admin_
...
+
+The "Reorder" button will not appear on the "My Orders" and "View Order" pages.
+
+![]({% link images/images/order-view-page.png %}){: .zoom} 
+
+## Cart isn’t empty 
+
+In case if Cart is not empty and user will click on 'Reorder' button (from “My Orders”  or “Order View” pages), the existing products will be added to the Shopping cart with the recorder products 
+
+![]({% link images/images/shopping-cart-view1.png %}){: .zoom}
```
Comment (atwixfirster) [inline src/sales/reorders-allow.md:78]: The same here. After `{: .zoom}`

File: src/sales/reorders-allow.md [hunk 6]
```diff
@@ -32,3 +32,61 @@ _Customer reorder link in the Admin_
...
+
+## Cart isn’t empty 
+
+In case if Cart is not empty and user will click on 'Reorder' button (from “My Orders”  or “Order View” pages), the existing products will be added to the Shopping cart with the recorder products 
+
+![]({% link images/images/shopping-cart-view1.png %}){: .zoom}  
+
+## Reorder from Admin Panel
+
+1. Open the Orders grid
```
Comment (atwixfirster) [inline src/sales/reorders-allow.md:82]: The same here.

File: src/sales/reorders-allow.md [hunk 7]
```diff
@@ -32,3 +32,61 @@ _Customer reorder link in the Admin_
...
+
+## Reorder from Admin Panel
+
+1. Open the Orders grid 
+
+2. Click the 'View' button
+
+“Reorder“ button is displayed on top menu.
+
+![]({% link images/images/order-view-admin.png %}){: .zoom}
```
Comment (atwixfirster) [inline src/sales/reorders-allow.md:88]: The same here. After `{: .zoom}`

File: src/sales/reorders-allow.md [hunk 8]
```diff
@@ -32,3 +32,61 @@ _Customer reorder link in the Admin_
...
+
+In case if Cart is not empty and user will click on 'Reorder' button (from “My Orders”  or “Order View” pages), the existing products will be added to the Shopping cart with the recorder products 
+
+![]({% link images/images/shopping-cart-view1.png %}){: .zoom}  
+
+## Reorder from Admin Panel
+
+1. Open the Orders grid 
+
+2. Click the 'View' button
```
Comment (atwixfirster) [inline src/sales/reorders-allow.md:84]: Please use `1`:

```
1. Click the 'View' button
```

**Review by atwixfirster — CHANGES_REQUESTED** [review]

**Review by atwixfirster — APPROVED** [review]
Thank you for your contributions, @EvheniiaKolkhiieva !

Approve from my side ✅

**Comment by atwixfirster** [comment]
@dobooth , could you please add the next labels to this PR:

- `partners-contribution`
- `Partner: Atwix`

Thank you!

File: src/sales/reorders-allow.md [hunk 9]
```diff
@@ -32,3 +32,61 @@ _Customer reorder link in the Admin_
...
+
+In case if Cart is not empty and user will click on 'Reorder' button (from “My Orders”  or “Order View” pages), the existing products will be added to the Shopping cart with the recorder products
+
+![]({% link images/images/shopping-cart-view1.png %}){: .zoom}
+
+## Reorder from Admin Panel
+
+1. Open the Orders grid
+
+1. Click the 'View' button
```
Comment (leslietilling) [inline src/sales/reorders-allow.md:84]:
```suggestion
1. Click **View**.
```

File: src/sales/reorders-allow.md [hunk 10]
```diff
@@ -32,3 +32,61 @@ _Customer reorder link in the Admin_
...
+
+“Reorder“ button is displayed on top menu.
```
Comment (leslietilling) [inline src/sales/reorders-allow.md:86]:
```suggestion
   _Reorder_ is displayed in the top button bar.
```

File: src/sales/reorders-allow.md [hunk 11]
```diff
@@ -32,3 +32,61 @@ _Customer reorder link in the Admin_
...
+
+![]({% link images/images/order-view-admin.png %}){: .zoom}
+
+After clicking the "Reorder" button the "Create New Order"page with reorder products will open.
```
Comment (leslietilling) [inline src/sales/reorders-allow.md:90]:
```suggestion
   After you click **Reorder**, the _Create New Order_ page opens with reorder products.
```

File: src/sales/reorders-allow.md [hunk 12]
```diff
@@ -32,3 +32,61 @@ _Customer reorder link in the Admin_
...
+
+## Reorder from Admin Panel
+
+1. Open the Orders grid
+
+1. Click the 'View' button
+
+“Reorder“ button is displayed on top menu.
+
+![]({% link images/images/order-view-admin.png %}){: .zoom}
```
Comment (leslietilling) [inline src/sales/reorders-allow.md:88]:
```suggestion
   ![]({% link images/images/order-view-admin.png %}){: .zoom}
```

File: src/sales/reorders-allow.md [hunk 13]
```diff
@@ -32,3 +32,61 @@ _Customer reorder link in the Admin_
...
+
+After clicking the "Reorder" button the "Create New Order"page with reorder products will open.
+
+![]({% link images/images/create-reorder-page.png %}){: .zoom}
```
Comment (leslietilling) [inline src/sales/reorders-allow.md:92]:
```suggestion
   ![]({% link images/images/create-reorder-page.png %}){: .zoom}
```

File: src/sales/reorders-allow.md [hunk 14]
```diff
@@ -32,3 +32,61 @@ _Customer reorder link in the Admin_
    No|Disables reorders.
 
 1. Click <span class="btn">Save Config</span>.
+
+## Reorder from Storefront
+
+Customer may initiate the reorder functionality for a particular order from the
```
Comment (leslietilling) [inline src/sales/reorders-allow.md:38]:
```suggestion
A customer can initiate the reorder functionality for a specific order from two pages:
```

**Review by leslietilling — COMMENTED** [review]
Edits for grammar, style, and formatting

**Comment by EvheniiaKolkhiieva** [comment]
Hi, @leslietilling,
Thank you for your reviewing!

I made some changes to the request, could you please check?

Thank you!

**Review by leslietilling — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1105`.

**Pull request merged**